react/features/authentication/middleware.js

```diff
@@ -108,9 +108,12 @@ MiddlewareRegistry.register(store => next => action => {
     case CONNECTION_FAILED: {
         const { error } = action;
 
-        error
+        if (error
             && error.name === JitsiConnectionErrors.PASSWORD_REQUIRED
-            && store.dispatch(_openLoginDialog());
+            && typeof error.recoverable === 'undefined') {
+            error.recoverable = true;
```

Comment:
style nit: when the if takes mutliple lines, the body needs one extra level of indentation

Reply:
ESLint will complain